src/lib/mongoBackend/mongoRegistrationCreate.cpp

```diff
@@ -211,7 +211,10 @@ void mongoRegistrationCreate
   setServicePath(servicePath, &bob);
   setContextRegistrationVector(regP, &bob);
   setStatus(regP->status, &bob);
-  setFormat("JSON", &bob);   // FIXME #3068: this would be unhardired when we implement NGSIv2-based forwarding
+
+  // FIXME #3068: this must be unhardwired once we fully implement NGSIv2-based forwarding (not only "normalized")
```

Comment:
I understand this comment should be removed. This PR has done the unhardwiring.

Reply:
Not really, now it is either "JSON" or "normalized"

Reply:
I wonder if it makes sense to implement other NGSIv2 formats appart from normalized for forwarding...

Reply:
Oh yes, it makes sense, but let's do that in another PR

Reply:
I don't think so. Extra formats beyond normalized (keyValues, etc.) are a way of givin flexibility to the final user. So, they make sense in operations done by the client.

However, CB-CPr is an internal communication in the platform so I don't see the need of adding extra complexity.

Whatever the case, the FIXME can be removed. The FIXME was about implementing any kind of NGSIv2 based format. Once we have one (normalized), it can be removed.

Reply:
fixed in c522e4eaf6d7bb140f7f12c7815049bcd23ff200